.github/PULL_REQUEST_TEMPLATE/infra.md

```diff
@@ -0,0 +1,35 @@
+# Description of change
+
+Please write a summary of your changes and why you made them.
+
+## Links to any relevant issues
+
+Be sure to reference any related issues by adding `fixes #(issue)`.
+
+## How the change has been tested
+
+Describe the tests that you ran to verify your changes.
+
+Make sure to provide instructions for the maintainer as well as any relevant configurations.
+
+- [ ] Basic tests (linting, compilation, formatting, unit/integration tests)
+- [ ] Patch-specific tests (correctness, functionality coverage)
+
+### Infrastructure QA (only required for crates that are maintained by @iotaledger/infrastructure)
```

Comment:
Probably don't need to mention that it's for infra only in here anymore right?

Reply:
@iotaledger/infrastructure can refine how they want the wording once this PR is merged